src/posts/us-exit-documents/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: "Repatriating documents: replacing Google Docs and Office 365 with European services"
+description: "Exploring, and selecting, European alternatives to Google Docs and Microsoft Office 365"
+date: 2025-03-24
+thumb: "anete-lusina-rFKBUwLg_WQ-unsplash.jpg"
+thumbCaption: "Photo by Anete Lūsiņa via Unsplash"
+thumbCaptionHref: "https://unsplash.com/photos/man-standing-inside-airport-looking-at-led-flight-schedule-bulletin-board-rFKBUwLg_WQ"
+tags:
+  - eurostack
+  - data-sovereignty
+---
+
+
+{% collapsible "TL;DR (spoilers)" %}
+Both me and factor10 went with [kDrive from Infomaniak](https://www.infomaniak.com/en/ksuite/kdrive).
+It has feature parity with both OneDrive and Google Drive (except for the photo handling of OneDrive) and
+uses [OnlyOffice](https://www.onlyoffice.com/) as document editors.
+
+I pay EUR 11.11 per month for 3 TiB storage and 6 users.
+
+factor10 pays EUR 6.58 per month per user for [kSuite pro](https://www.infomaniak.com/en/ksuite/ksuite-pro),
+which includes 3 TiB of storage per user.
+That also includes the rest of kSuite,
+if you go with just kDrive Pro it's EUR 7.40 per month but includes 6 TiB of storage per user.
+{% endcollapsible %}
+
+I've been dissatisfied with most of the "free" services that we use day-to-day for quite a while now.
+I shut down my Facebook account in 2018 and abandoned my Instagram account not too long after.
+I've never used GMail, and I've been using alternative search engines for almost a decade now
+(though these are also free, and unfortunately often source their results from either Google or Microsoft).
+
+Another aspect is GDPR.
+The EU-US Data Privacy Framework is built on very shaky grounds, and it is going to be challenged in court on April 1
+&mdash; and I think it is likely to fall.
+
+At factor10, we started talking about replacing all our US-based services with european alternatives because of GDPR.
+While there has always been some sort of framework or agreement int place between the EU and the US,
+they have always fallen, and we're surprised that the current one has held up this long.
+
+We never really did get off the ground with this and never replaced anything.
+
+Fast-forward until mid-January 2025 and suddenly it has become a priority.
+
+# What we're replacing.
+
+## Me
+
+Personally, I've had all my files in OneDrive for quite a while.
+When my wife entered the picture we got an Office 365 family account and moved all her files from iCloud to OneDrive 
+so we could keep track of everything together.
+
+When I acquired a server running TrueNAS, I set it up with bidirectional sync via RClone 
+&mdash; this has unfortunately not worked for a couple of months because OneDrive seems to tell RClone some files have different sizes than what is then sent,
+causing RClone to raise a data validation error.
+
+## factor10
+
+We have all our files in Google Drive - about 250 GiB of data.
```

Comment:
I think it's good to mention that we have our customer's data in our customer's solutions and that it's our internal files we have at Google Drive.

I prefer skipping to mention the amount of data. It might sound like a lot, but it's videos and photos that are "driving" the size.

Reply:
Jag förväntade mig att det var betydligt mer och blev förvånad att det är så lite när Per kollade :)

Men jag tar bort siffran